src/davinci/models.py

```diff
@@ -1,3 +1,16 @@
-# from django.db import models
+"""
+Models for Da Vinci mission.
+"""
+import datetime
+from django.db import models
+from ewilgs.models import Downlink
 
-# Create your models here.
+class DaVinci_L0_telemetry(models.Model): #pylint: disable=C0103
+    """
+    Telemetry Table Delfin3xt
```

Comment:
Da Vinci. Take care when copy pasting, can propagate bugs... I learnt the hard way :)